src/main/java/ch/njol/skript/expressions/ExprCommand.java

```diff
@@ -36,33 +37,31 @@
 import ch.njol.skript.lang.util.SimpleExpression;
 import ch.njol.util.Kleenean;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Command")
 @Description("The command that caused an 'on command' event (excluding the leading slash and all arguments)")
-@Examples({"# prevent any commands except for the /exit command during some game",
-		"on command:",
-		"\tif {game::%player%::playing} is true:",
-		"\t\tif the command is not \"exit\":",
-		"\t\t\tmessage \"You're not allowed to use commands during the game\"",
-		"\t\t\tcancel the event"})
-@Since("2.0, INSERT VERSION (support for script commands)")
-@Events("command")
+@Examples({
+	"# prevents any commands except for the /exit command during some game",
+	"on command:",
+	"\tif {game::%player%::playing} is true:",
+	"\t\tif the command is not \"exit\":",
+	"\t\t\tmessage \"You're not allowed to use commands during the game\"",
+	"\t\t\tcancel the event"})
+@Since("2.0, INSERT VERSION (support for script commands, unknown command event)")
+@Events({"command", "unknown command"})
 public class ExprCommand extends SimpleExpression<String> {
 
 	static {
 		Skript.registerExpression(ExprCommand.class, String.class, ExpressionType.SIMPLE,
-				"[the] (full|complete|whole) command",
-				"[the] command [(label|alias)]"
+			"[the] (full|complete|whole) command",
+			"[the] command [(label|alias)]"
 		);
 	}
 
 	private boolean fullCommand;
 	
 	@Override
 	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
-		if (!getParser().isCurrentEvent(PlayerCommandPreprocessEvent.class, ServerCommandEvent.class, ScriptCommandEvent.class)) {
+		if (!getParser().isCurrentEvent(PlayerCommandPreprocessEvent.class, ServerCommandEvent.class, ScriptCommandEvent.class, UnknownCommandEvent.class)) {
```

Comment:
This is a Paper exclusive event. Needs a Skript.classExists method check. Typically for init methods, we put a private static final boolean field in the class that defines if the class exists for usage.

Reply:
Just to double check before I finalize anything this is the format of how it should be structured correct.
*the new line is just to make showcasing easier
![image](https://user-images.githubusercontent.com/81952476/210126604-e78495cb-c314-41c3-a044-34883d4bcd3f.png)

Reply:
the one is related to above missed placed it woops